Sprint-1/implement/max.js

```diff
@@ -1,4 +1,80 @@
+let a = [3, 5, 9, 12];
```

Comment:
It is never too early to develop the habit to keep the code clean by removing 
(or at least commenting out) unused code or debugging code.